ddionrails/imports/management/commands/update.py

```diff
@@ -1,36 +1,57 @@
-from pathlib import Path
+# -*- coding: utf-8 -*-
+""" Update management command for ddionrails project """
+
+import pathlib
 
 import djclick as click
+import git
 from django.conf import settings
-from git import Repo
 
 from ddionrails.studies.models import Study
 
 
-def update_study(study):
-    repo_dir = Path(settings.IMPORT_REPO_PATH) / study.name
-    if repo_dir.exists():
-        print("pulling")
-        repo = Repo(repo_dir)
-        repo.remotes.origin.pull(depth=1)
+def update_study(study: Study) -> None:
+    """ Clone a study's git repository with --depth=1
+        or pull changes if git repository has already been cloned.
+
+        The path to the git repository /repositories is set via the
+        settings.IMPORT_REPO_PATH variable.
+    """
+    repository_directory = pathlib.Path(settings.IMPORT_REPO_PATH) / study.name
+    if repository_directory.exists():
+        print(f'Pulling "{study.name}" from "{study.repo_url()}"')
+        repo = git.Repo(repository_directory)
+        repo.remotes.origin.pull()
```

Comment:
This line changes the behaviour of the pulling from
` repo.remotes.origin.pull(depth=1)` to  `repo.remotes.origin.pull()`.
In case there is too much going on in the diff.